From f352503ed2968279b2ffa8d9730f7d27e9ea5b36 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sat, 27 Mar 2021 17:21:36 +0100
Subject: [PATCH 1/4] - Add shortcuts for Software Center in menu

- Add shortcuts for Software Center in menu
- Add solus icon to kicker menu

---
 applets/kicker/package/contents/config/main.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applets/kicker/package/contents/config/main.xml b/applets/kicker/package/contents/config/main.xml
index 0141730..5123578 100644
--- a/applets/kicker/package/contents/config/main.xml
+++ b/applets/kicker/package/contents/config/main.xml
@@ -38,7 +38,7 @@
     </entry>
     <entry name="favoriteApps" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover</default>
+      <default>preferred://browser,systemsettings.desktop,org.kde.dolphin.desktop,org.kde.konversation.desktop,,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoriteSystemActions" type="StringList">
       <label>List of system action favorites.</label>
-- 
2.30.2

